sdk/servicebus/azure-servicebus/tests/stress_tests/stress_test_base.py

```diff
@@ -132,8 +142,11 @@ def _Send(self, sender, end_time):
                 while end_time > datetime.utcnow():
                     print("SENDING")
                     message = self._ConstructMessage()
-                    sender.send_messages(message)
-                    self.OnSend(self._state, message)
+                    if self.send_session_id != None:
+                        sender.send_messages(message, session_id=self.send_session_id)
```

Comment:
I don't think our `send_message` accept `session_id`.

should be:
```
msg = Message(session_id='xxx')
# or
msg = Message(...)
msg.session_id = 'xxx'
```

Reply:
You're correct.  I can only assume I was on autopilot and thinking about T1.  (Something something "this is why I rant about kwargs being a pitfall")